src/components/Categories.js

```diff
@@ -0,0 +1,8 @@
+import React from 'react';
+import Button from './Button';
+
+const Categories = () => {
+  <Button text="Check Status" id="status" buttonType="button" />;
```

Comment:
- [ ] <b>REQUIRED:</b> Kindly make sure this button show text Check status.  Because currently,  it is not showing anything.
![Screen Shot 2022-04-02 at 06 50 14](https://user-images.githubusercontent.com/45217364/161366832-54dc7724-b94f-4dc5-9b2a-f32c4e32893b.png)

Here is the way to fix that.
```js
<Button id="status" buttonType="button"> Check Status </button>
```
And also, in the App component, remember to render `form` component as Well.

Reply:
Thanks for the review. I have rendered the form component in the App component. But I do not see any `check status` on the project photo, so I cannot understand what you are saying. Please come clear

Reply:
Thank you for the review! I have rendered the form component in the App component. But there is no button that shows the text "Check Status" on the project template. Here is a reminder, please.
`
![bookstore](https://user-images.githubusercontent.com/92922987/161368154-c81006e8-7745-418a-9904-7b60d5c32f87.png)
`

Reply:
@rukundoeric  Please consider the comment above